code/modules/antagonists/traitor/medrp/new_objectives.dm

```diff
@@ -0,0 +1,36 @@
+
+
+/obj/effect/proc_holder/spell/self/add_objectives //If all of your objectives are done, you get two more.
```

Comment:
THIS SHOULD NOT BE A SPELL, ABSOLUTELY DISGRACEFUL, LEARN HOW TO ADD ACTION BUTTONS